src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -60,7 +59,8 @@ public Command setCenterRollerSpeed(MutableMeasure<Velocity<Angle>> speed) {
     public void periodic() {
         io.updateInputs();
         Logger.processInputs(this.getClass().getSimpleName(), inputs);
-        Logger.recordOutput("IntakePose", new Pose3d(new Translation3d(0,0,0), new Rotation3d(0,inputs.currentAngle.in(Units.Degrees),0)));
+        Logger.recordOutput("IntakePose", new Pose3d(new Translation3d(0.327,0,0.165), new Rotation3d(0,-inputs.currentAngle.in(Units.Radians),0)));
```

Comment:
I think it's better to set angle 0 at the folded position. That way the direction will be as AdvantageScope's CCW+.

Reply:
CCW+ means "counter-clockwise is positive". Currently, the zero position is the open position (when the intake is parallel to the ground), so to match AdvantageScope's CCW+ direction, the folded position needs to be negative. You can instead set the folded position to be the zero position and then the open position will be 90°. That way you won't need to deal with negative numbers.